table/block_based/block_based_table_reader_test.cc

```diff
@@ -224,7 +224,12 @@ TEST_P(BlockBasedTableReaderTest, MultiGet) {
 INSTANTIATE_TEST_CASE_P(
     MultiGet, BlockBasedTableReaderTest,
     ::testing::Combine(::testing::ValuesIn(GetSupportedCompressions()),
-                       ::testing::Bool()));
+#ifdef ROCKSDB_LITE
+                       ::testing::Values(false)
+#else   // ROCKSDB_LITE
+                       ::testing::Bool()
+#endif  // ROCKSDB_LITE
+                           ));
```

Comment:
for some reason windows compiler doesn't like this.
```
         C:\Users\circleci\project\table\block_based\block_based_table_reader_test.cc(232,1): error C2121: '#': invalid character: possibly the result of a macro expansion [C:\Users\circleci\project\build\rocksdb_block_based_table_reader_test.vcxproj]
```

Reply:
This is a macro in parameter in macro.... Too complicated.